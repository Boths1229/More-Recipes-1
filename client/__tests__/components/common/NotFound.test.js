import React from 'react';
import { shallow } from 'enzyme';
import NotFound from '../../../components/common/NotFound';

describe('NotFound Component', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<NotFound />);
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find('header').length).toBe(1);
    expect(wrapper.find('img').length).toBe(1);
    expect(wrapper.find('h1').length).toBe(1);
  });
});
