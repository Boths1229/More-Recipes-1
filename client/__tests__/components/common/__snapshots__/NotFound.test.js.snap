// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound Component should render correctly 1`] = `
<div>
  <header>
    <nav
      className="navbar fixed-top navbar-expand-lg navbar-dark"
      style={
        Object {
          "backgroundColor": "#2b3034",
        }
      }
    >
      <a
        className="navbar-brand"
        href="/"
      >
        More-Recipes
      </a>
      <button
        aria-controls="navbarSupportedContent"
        aria-expanded="false"
        aria-label="Toggle navigation"
        className="navbar-toggler"
        data-target="#navbarSupportedContent"
        data-toggle="collapse"
        type="button"
      >
        <span
          className="navbar-toggler-icon"
        />
      </button>
      <div
        className="collapse navbar-collapse"
        id="navbarSupportedContent"
      >
        <ul
          className="navbar-nav ml-auto"
        >
          <li
            className="nav-item"
          >
            <Link
              className="nav-link navbar-brand"
              onlyActiveOnIndex={false}
              style={Object {}}
              to="/recipes"
            >
              Home
            </Link>
          </li>
        </ul>
      </div>
    </nav>
  </header>
  <div
    className="container main"
  >
    <div
      className="pt-5 main-div"
      style={
        Object {
          "textAlign": "center",
        }
      }
    >
      <img
        alt="error"
        className="img-fluid rounded-circle"
        name="image"
        src="http://clipartmag.com/images/oops-emoticon-17.jpg"
      />
      <h1>
        Page Not Found
      </h1>
      <div>
        <Link
          className="vote-button errorBtn"
          id="remove-link"
          onlyActiveOnIndex={false}
          style={Object {}}
          to="/recipes"
        >
          Go Back
        </Link>
      </div>
    </div>
  </div>
  <Footer />
</div>
`;
